adoc-mode.el

```diff
@@ -123,6 +124,38 @@ need to call `adoc-calc' for the change to take effect."
                  function)
   :group 'adoc)
 
+(defcustom adoc-hide-markup nil
```

Comment:
I think you copied this one from Markdown mode, but it's never used anywhere in your PR.

Reply:
Removed.